src/Npgsql/NpgsqlTransaction.cs

```diff
@@ -46,10 +46,9 @@ public sealed class NpgsqlTransaction : DbTransaction
         internal bool IsDisposed;
 
         /// <summary>
-        /// Specifies the <see cref="System.Data.IsolationLevel">IsolationLevel</see> for this transaction.
+        /// Specifies the isolation level for this transaction.
         /// </summary>
-        /// <value>The <see cref="System.Data.IsolationLevel">IsolationLevel</see> for this transaction.
-        /// The default is <b>ReadCommitted</b>.</value>
+        /// <value>The isolation level for this transaction. The default is <see cref="System.Data.IsolationLevel.ReadCommitted"/>.</value>
```

Comment:
```suggestion
        /// <value>The isolation level for this transaction. The default is <see cref="IsolationLevel.ReadCommitted"/>.</value>
```

Reply:
The namespace is needed here because otherwise IsolationLevel resolved to the property on this class.

Reply:
(more precisely, it doesn't compile because of ambiguity...)

Reply:
Ah, I see. Then you can try to prefix it with `T:`:
```suggestion
        /// <value>The isolation level for this transaction. The default is <see cref="T:IsolationLevel.ReadCommitted"/>.</value>
```

Reply:
That's true, but unless I'm mistaken that means you lose the Intellisense/refactoring/ability to navigate to it, no?

Reply:
Don't know that detail. It needs to be checked.